source/Ship.cpp

```diff
@@ -1490,8 +1491,10 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	DoGeneration();
 
 	// Handle ionization effects, etc.
-	if(ionization)
+	if(ionization || scrambling)
 		CreateSparks(visuals, "ion spark", ionization * .1);
+	if(scrambling)
+		CreateSparks(visuals, "scrambling spark", scrambling * .1);
```

Comment:
Should probably half the amount of each spark type if the plan is to always pair them.
```suggestion
		CreateSparks(visuals, "ion spark", ionization * .05);
	if(scrambling)
		CreateSparks(visuals, "scrambling spark", scrambling * .05);
```

Reply:
Thats what I had done at first but then I did the same as the other ones to not have an exception

Reply:
Well then we'd be creating twice as many sparks with every current ion weapon, which I don't think is what we want.